src/client/components/terminal/share.ts

```diff
@@ -8,13 +8,22 @@ import { SaveIcon } from '../icons/save'
 @customElement('share-cell')
```

Comment:
Is this still an appropriate name for the component?

Reply:
Sure, I think it should be more generic. Let me rename it.

Reply:
Something like `ActionButton` might be a good choice